staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -159,6 +159,7 @@ func NewCmdLogs(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.C
 			cmdutil.CheckErr(o.RunLogs())
 		},
 	}
+	cmd.AddCommand(NewCmdNodeLogs(f, streams))
```

Comment:
As with every alpha command add it to https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/alpha.go for now, we'll decide about it future location when this moves to beta.